src/proto/nc_redis.c

```diff
@@ -1680,6 +1712,20 @@ redis_parse_req(struct msg *r)
                 r->state);
 }
 
+static int update_and_check_done(struct msg *r) {
```

Comment:
all functions in twemproxy must be namspaced like redis_ 

also the return value must be either rstatus_t or bool